common/skiplinks/reports/ally-1-en.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Skip links",
+	"language": "en",
+	"description": "Evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
+	"tag": "skiplinks",
+	"parentdir": "skiplinks",
+	"parentPage": "Skip links",
+	"parentPageURL": "skiplinks",
+	"altLangPage": "a11y-1-fr.html",
+	"dateModified": "2023-07-06",
+	"layout": "assessment_wrote_en-en",
+	"reportURL": "pre-a11y-1.json"
```

Comment:
```suggestion
	"reportURL": "pre-ally-1.json"
```

Reply:
switched all to use a11y for consistency